client/signup/steps/about/index.jsx

```diff
@@ -254,23 +254,25 @@ class AboutStep extends Component {
 		const siteTitleInput = formState.getFieldValue( this.state.form, 'siteTitle' );
 		const siteGoalsInput = formState.getFieldValue( this.state.form, 'siteGoals' );
 		const siteGoalsArray = siteGoalsInput.split( ',' );
+		const siteGoalsGroup = siteGoalsArray.sort().join();
 		const userExperienceInput = this.state.userExperience;
 		const siteTopicInput = formState.getFieldValue( this.state.form, 'siteTopic' );
 
 		//Site Title
 		if ( siteTitleInput !== '' ) {
 			siteTitleValue = siteTitleInput;
 			this.props.setSiteTitle( siteTitleValue );
-			this.props.recordTracksEvent( 'calypso_signup_actions_user_input', {
-				field: 'Site title',
-				value: siteTitleInput,
-			} );
 		}
 
+		this.props.recordTracksEvent( 'calypso_signup_actions_user_input', {
+			field: 'Site title',
+			value: siteTitleInput !== '' ? siteTitleInput : 'N/A',
```

Comment:
not important, but we should also be able to shorten to `value: siteTitleInput || 'N/A'`